railgun/test/integration/suite_test.go

```diff
@@ -66,6 +66,9 @@ var (
 
 	// cluster is the object which contains a Kubernetes client for the testing cluster
 	cluster ktfkind.Cluster
+
+	// dbmode indicates the database backend of the test cluster ("off" and "postgres" are supported)
+	dbmode = os.Getenv("KONG_DATABASE")
```

Comment:
Ah, just realized that the `KONG_` prefix is _de facto_ used by gateway. The controller uses `CONTROLLER_`. If you want to define vars that are to be consumed by the test, you probably want something other than these two.

Reply:
Sounds good, 7acd35ca8acb85ddbbcaa0688bfaa727fc5aa689 resolves :+1: